public/data/nirbhay12345.json

```
@@ -0,0 +1,18 @@
+{
+    "name": "Nirbhay Chaplot",
+    "type": "personal",
+    "bio": "Open Source Enthusiast!",
+    "avatar": "https://github.com/nirbhay12345.png",
+    "links": [
+        {
+            "name": "Follow me on GitHub",
+            "url": "https://github.com/nirbhay12345",
+            "icon": "github"
+        },
+        {
+            "name": "Follow me on Twitter",
+            "url": "https://twitter.com/Nirbhay00704167",
+            "icon": "twitter"
+        }
+    ]
+}
```

Missing an end line.

```suggestion
}

```

---

I have done the changes